apps/src/code-studio/pd/form_components/button_list.jsx

```diff
@@ -20,7 +26,6 @@ const ButtonList = React.createClass({
 
   handleChange(event) {
     let value;
-    console.log(event);
```

Comment:
We absolutely should! looks like we currently have over a hundred violations of [no-console](http://eslint.org/docs/rules/no-console), so it at the very least warrants its own PR.